Infomondia

vgrunseid opened the pull request:
Creacion de etl para levantar el  informe monetario diario del BCRA, archivo infomondia.xls. Consta de dos archivos:
dll.sql: script de creacion de tablas
downloadInfomondia.py : codigo  que baja, procesa y levanta datos en las tablas creadas en dll.sql

vgrunseid committed:
creacion

vgrunseid committed:
incluyo vars.sh, DS_Store  en .gitignore

jmtruffa commented:
Está muy bien y se ve que pudiste hacer el PR sin problemas.

Estuve mirando el código y me pregunto por qué haces un delete inicial de los contenidos de la base?
Podrías perfectamente esa línea no tenerla, y cambiar al momento del grabado del df poniendo un parametro: if_exists='replace' en la llamada nombre-del-dataframe.to_sql():

dataframe.to_sql(..., if_exists='replace', ...)

Puede que no sea mucho más rápido, pero me parece que el código puede ser más prolijo.

vgrunseid committed:
se eliminaron los deletes de las tablas destino por redundancia

vgrunseid reviewed (commented):
Eliniados los delete de las tablas destino por ser redundantes

jmtruffa committed:
Delete vars.sh

No es necesario tener las variables de entorno en el repo. Deben quedar locales.

jmtruffa commented:
Ok, mando el merge
